src/Core/Form/IdentifiableObject/CommandBuilder/Product/ProductSuppliersCommandsBuilder.php

```diff
@@ -40,36 +41,45 @@ final class ProductSuppliersCommandsBuilder implements ProductCommandsBuilderInt
      */
     public function buildCommands(ProductId $productId, array $formData): array
     {
-        if (!isset($formData['options']['suppliers']['product_suppliers']) && !isset($formData['options']['suppliers']['default_supplier_id'])) {
+        if (!isset($formData['options']['suppliers']['supplier_ids']) &&
```

Comment:
Are you agree to say that this expression can be difficult to understand. In clean code is recommanded to avoid multiple conditions especially when they use the not option.
For the future reader may be we can move into a private method with an explicit name ?